packages/server/src/notification/notification.controller.ts

```diff
@@ -39,11 +51,23 @@ export class NotificationController {
 
   // TODO: GET RID OF THIS
   @Post('/notify_user/:user_id')
+  @UseGuards(JwtAuthGuard)
   async notifyUser(@Param('user_id') user_id: number): Promise<string> {
     await this.notifService.notifyUser(
       user_id,
       "bruh if you're still tryna test notifications, ligma balls, FUCK FUCK FUCK FUCK FUCK, sugondese, hot cheetos",
     );
     return 'notified';
   }
+
+  @Post('/phone/verify')
+  async verifyPhoneUser(
```

Comment:
Also I think we could add a comment making it more obvious this is a webhook endpoint that comes from twilio and linking their docs. Also I'm wondering if twilio has a way for us to verify their identity? What's stopping me from postmanning this endpoint with Alex's phone number?